docs/typescript/data-converters.md

```diff
@@ -0,0 +1,208 @@
+---
+id: data-converters
+title: Data Converters
```

Comment:
Just a few small changes and this is close to "How to use a custom Data Converter in TypeScript"